products/kubernetes-cluster-autoscaler.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Kubernetes Cluster Autoscaler
+category: server-app
+tags: kubernetes
+permalink: /cluster-autoscaler
+versionCommand: |-
+  cluster-autoscaler --version
+releasePolicyLink: https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler#releases
+changelogTemplate: https://github.com/kubernetes/autoscaler/releases/tag/cluster-autoscaler-__LATEST__
+releaseDateColumn: true
+eolColumn: true
+
```

Comment:
```suggestion

auto:
  methods:
  -   git: https://github.com/kubernetes/autoscaler.git
      regex: ^cluster-autoscaler-(?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)$
```

So that latest versions are automatically updated. We could also use the github release date instead, what do you think is the best ?